lhotse/cut.py

```diff
@@ -299,7 +324,7 @@ def from_dict(data: dict):
         raw_cut = data.pop('cut')
         try:
             cut = Cut.from_dict(raw_cut)
-        except:
+        except TypeError:
```

Comment:
Are you sure it's going to be a `TypeError`? I admit I was lazy and did not check when writing this, and the change is in the right direction, just double-checking that this does not break loading PaddingCuts

Reply:
By analyzing `Cut.from_dict()`, I believe the expected exception type here should be `TypeError`. And according the following codes:
```
try:
    cut = Cut.from_dict(raw_cut)
except TypeError:
    cut = PaddingCut.from_dict(raw_cut)
```
It shows that once an exception raised, `PaddingCut.from_dict()` would be called instead of `Cut.from_dict()`, which implies the type of `raw_cut` does not match `Cut` but matches `PaddingCut`.